modules/misc/qt.nix

```diff
@@ -206,9 +208,10 @@ in {
     # Apply theming also to apps started by systemd.
     systemd.user.sessionVariables = envVars // envVarsExtra;
 
-    home.packages = (platformPackages.${cfg.platformTheme} or [ ])
-      ++ lib.optionals (cfg.style.package != null)
-      (lib.toList cfg.style.package);
+    home.packages = (lib.optionals (cfg.platformTheme != null)
+      platformPackages.${cfg.platformTheme})
+      ++ (lib.optionals (cfg.style.package != null)
+        (lib.toList cfg.style.package));
 
     xsession.importedVariables = [ "QT_PLUGIN_PATH" "QML2_IMPORT_PATH" ]
```

Comment:
This wasn't added in this PR but might as well ask here. Why do we need to set this if we're already importing these variables into the systemd user session above? AFAIU all that this does is to manually import the variables into systemd on xsession start (https://github.com/nix-community/home-manager/blob/master/modules/xsession.nix#L186-L188).